spec/services/event_log/move_runner_spec.rb

```diff
@@ -114,6 +114,40 @@
     end
   end
 
+  context 'when event_name=reject' do
+    let!(:event) { create(:move_event, :reject, eventable: move) }
+
+    context 'when the move is requested' do
```

Comment:
is it legitimate to reject a `requested` move? should the move be rejected only if it is `proposed`?

Reply:
leaving this question until we have a well-defined state machine